libvips/iofuncs/header.c

```diff
@@ -782,6 +802,18 @@ vips_image_guess_interpretation(const VipsImage *image)
 		return vips_image_default_interpretation(image);
 }
 
+/**
+ * vips_image_guess_max_alpha:
+ * @image: image to guess for
+ *
+ * Returns: the maximum alpha value for the image interpretation.
+ */
+double
+vips_image_guess_max_alpha(const VipsImage *image)
```

Comment:
I'd get rid of this, I don't think we need both.

Reply:
Wast't sure about this too. Removed